### ISSUE TITLE
Starting a new app with react-create-app does not work out of the box

### ISSUE THREAD

**Issue by jhvanderven** [emeeks/semiotic#49]
On Windows 10, I did:
```
npm i -g react-create-app
react-create-app myapp
cd myapp
npm i -S semiotic

```
Then I created the DivergingStackedBar component, modified App.js into:

```
import React, { Component } from 'react';
import DivergingStackedBar from './DivergingStackedBar';

import './App.css';

class App extends Component {
  render() {
    return (
      <DivergingStackedBar />
    );
  }
}

export default App;
```
And got errors on _react.PropTypes being undefined.
This seems to have changed since react v15.5.
To get the example working I had to:
```
npm remove react
npm i -SE react@15.4
npm remove react-dom
npm i -SE react-dom@15.4
npm start

```

**Comment by emeeks** [comment]
Thanks for the note, I didn't realize that Semiotic was so closely tied to 15.4 but 15.4.2 is the version referenced in the `package.json`, I just didn't realize the PropTypes had changed in between. I'm going to transition it to React 16 over the course of the next couple weeks but I'll see if I can clean this up in the interim.

**Comment by emeeks** [comment]
So I was digging into this and searching through my code I don't see anywhere where I'm using `React.PropTypes` I only see prop types being imported from `prop-types`. What version of Semiotic are you using and what was the exact error?

**Comment by jhvanderven** [comment]
This is the relevant part of package.json:

```
  "dependencies": {
    "d3-scale": "1.0.6",
    "d3-shape": "1.2.0",
    "d3-svg-legend": "2.24.1",
    "react": "15.4.2",
    "react-dom": "15.4.2",
    "react-scripts": "1.0.14",
    "semiotic": "^1.1.0"
  },

```
The exact error... I think it was on _React.PropTypes.number, where _React.PropTypes was undefined.

**Comment by emeeks** [comment]
I see the reason now, it's the `react-annotation` library that is still using React.PropTypes instead of `prop-types`. I'll file an issue and update and close this once that's sorted.

**Comment by susielu** [comment]
This should be resolved in version 0.0.1-4 please let me know if you have any issues with it.

**Comment by emeeks** [comment]
This should be resolved in the latest version.
